### PR TITLE
Fix 3D map toggle in side bar

### LINKED ISSUE #1273: Can not turn on 3D Preview with mouse click
### Description

If I use mouse to turn on checkbox to show 3D preview, nothing happens.
However if I use <kbd>CTRL</kbd>+<kbd>/</kbd> shortcut it works.
No errors in dev console that I can see.

Happens in Firefox, Chrome, Edge.

### Screenshots

image

### Version

2.2.1

### What browser are you seeing the problem on? What version are you running?

Chrome v121.0

### The OS you're using

win

### Steps to reproduce

1. Go to Background Settings
2. Click on "Show 3D Map"

### The browser URL at the time you encountered the bug

https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=20.00/40.74584/-73.98620

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36

### PR DESCRIPTION
**Pull request by tannerwuster**
There was an issue that if you check the 3D checkbox to show 3D map, nothing happens. This now works and fixes #1273

### CODE REVIEW

File: modules/ui/map3d_viewer.js [hunk 1]
```diff
@@ -277,7 +274,6 @@ export function uiMap3dViewer(context) {
     map.on('draw', () => redraw());
     map.on('move', () => redraw());
     context.keybinding().on([uiCmd('⌘' + l10n.t('background.3dmap.key'))], toggle);
-    urlhash.on('hashchange', toggle);
```
Comment (bhousel) [inline modules/ui/map3d_viewer.js:280]: My initial thought is that it would be a shame to lose the connection between the `map3d` urlhash param, and the state of the viewer, but looking elsewhere around the code it seems like this was never really set up correctly so we aren't losing anything.

I expect it to work like this:
- checking/unchecking the box updates the `map3d=true` to the urlhash  
(something like `urlhash.setParam('map3d', !_isHidden ? 'true' : null)`)
- changing the urlhash updates the state of the box and viewer
(something like `urlhash.on('hashchange', doStuff)`)

Basically we should just look at how `PhotoSystem` or `ImagerySystem` works and make `Map3dSystem` work more like those.

Anyway, I'm going to just merge this because we need to patch the bug, but we should improve this later.

**Review by bhousel — APPROVED** [review]
